lib/ex_format.ex

```diff
@@ -1,1371 +1,23 @@
 import Kernel, except: [to_string: 1]
+import ExFormat.Formatter, only: [to_string_with_comments: 1]
+import ExFormat.State
+import ExFormat.AST
+import ExFormat.Comments
+import ExFormat.Lines
```

Comment:
Just wondering - why is `import` considered a last resort? Is there any performance hits when using it?